src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/PortableInfoboxPageObject.java

```diff
@@ -96,7 +100,7 @@ public WebElement getGroupHeader(int index) {
   }
 
   public String getUrlFromExternalLinkaAfterPageIsLoaded() {
-    wait.forElementPresent(By.id("#footer"));
+    wait.forElementVisible(bodyElement);
```

Comment:
why are you waiting for body ? 
And I can't see usage of this method anywhere after, why are you modyfing it?